test_texi2rst.py

```diff
@@ -1229,6 +1229,34 @@ def test_table_with_variable_number_of_columns(self):
 ========  ====================================================================  =======  ===========  =============
 ''', out)
 
+    def test_table_with_empty_column(self):
+        xml_src = '''
+<multitable spaces=" " endspaces=" "><columnfractions line=".15 .30 .40"><columnfraction value=".15"></columnfraction><columnfraction value=".30"></columnfraction><columnfraction value=".40"></columnfraction></columnfractions>
+<tbody><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(1)</code></para></entry><entry command="tab" spaces=" "><para>The year
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(2)</code></para></entry><entry command="tab" spaces=" "><para>The month
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(3)</code></para></entry><entry command="tab" spaces=" "><para>The day of the month
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(4)</code></para></entry><entry command="tab" spaces=" "><para>Time difference with UTC in minutes
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(5)</code></para></entry><entry command="tab" spaces=" "><para>The hour of the day
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(6)</code></para></entry><entry command="tab" spaces=" "><para>The minutes of the hour
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(7)</code></para></entry><entry command="tab" spaces=" "><para>The seconds of the minute
+</para></entry></row><row><entry command="item" spaces=" "></entry><entry command="tab" spaces=" "><para><code>VALUE(8)</code></para></entry><entry command="tab" spaces=" "><para>The milliseconds of the second
+</para></entry></row></tbody></multitable>'''
+        tree = from_xml_string(xml_src)
+        tree = convert_to_rst(tree, self.ctxt)
+        out = self.make_rst_string(tree)
+        self.assertEqual('''  ============  ===================================
+  ``VALUE(1)``  The year
+  ============  ===================================
+  ``VALUE(2)``  The month
+  ``VALUE(3)``  The day of the month
+  ``VALUE(4)``  Time difference with UTC in minutes
+  ``VALUE(5)``  The hour of the day
+  ``VALUE(6)``  The minutes of the hour
+  ``VALUE(7)``  The seconds of the minute
+  ``VALUE(8)``  The milliseconds of the second
+  ============  ===================================
+''', out)
+
```

Comment:
This seems to be from `fortran/intrinsic.texi` which has:
```
@var{VALUES} is @code{INTENT(OUT)} and provides the following:

@multitable @columnfractions .15 .30 .40
@item @tab @code{VALUE(1)}: @tab The year
@item @tab @code{VALUE(2)}: @tab The month
@item @tab @code{VALUE(3)}: @tab The day of the month
@item @tab @code{VALUE(4)}: @tab Time difference with UTC in minutes
@item @tab @code{VALUE(5)}: @tab The hour of the day
@item @tab @code{VALUE(6)}: @tab The minutes of the hour
@item @tab @code{VALUE(7)}: @tab The seconds of the minute
@item @tab @code{VALUE(8)}: @tab The milliseconds of the second
@end multitable

```

Looking at the generated `.rst`, the `VALUE(1)` row seems to be being used as a heading row and that seems wrong.  Is there a way to fix that, and have the table not have a heading row?  (and somehow sense this case, as opposed to other tables that should have such headings)

Reply:
You are right, it's being used as heading, that's another story.
I think root cause is wrong .texi as you correctly pointed out. I'm going to fix it upstream:

```patch
diff --git a/gcc/fortran/intrinsic.texi b/gcc/fortran/intrinsic.texi
index 73baa34104e..c74a7f56c60 100644
--- a/gcc/fortran/intrinsic.texi
+++ b/gcc/fortran/intrinsic.texi
@@ -4764,15 +4764,15 @@ Unavailable time and date parameters return blanks.
 
 @var{VALUES} is @code{INTENT(OUT)} and provides the following:
 
-@multitable @columnfractions .15 .30 .40
-@item @tab @code{VALUE(1)}: @tab The year
-@item @tab @code{VALUE(2)}: @tab The month
-@item @tab @code{VALUE(3)}: @tab The day of the month
-@item @tab @code{VALUE(4)}: @tab Time difference with UTC in minutes
-@item @tab @code{VALUE(5)}: @tab The hour of the day
-@item @tab @code{VALUE(6)}: @tab The minutes of the hour
-@item @tab @code{VALUE(7)}: @tab The seconds of the minute
-@item @tab @code{VALUE(8)}: @tab The milliseconds of the second
+@multitable @columnfractions .15 .70
+@item @code{VALUE(1)}: @tab The year
+@item @code{VALUE(2)}: @tab The month
+@item @code{VALUE(3)}: @tab The day of the month
+@item @code{VALUE(4)}: @tab Time difference with UTC in minutes
+@item @code{VALUE(5)}: @tab The hour of the day
+@item @code{VALUE(6)}: @tab The minutes of the hour
+@item @code{VALUE(7)}: @tab The seconds of the minute
+@item @code{VALUE(8)}: @tab The milliseconds of the second
 @end multitable
 
 @item @emph{Standard}:
```